release/tools/jta.xml

```diff
@@ -22,7 +22,7 @@
     <import file="../../bin/xml/ts.common.props.xml"/>
     
     <property name="deliverable.version" value="2.0"/>
-    <property name="deliverable.tck.version" value="2.0.2"/>
+    <property name="deliverable.tck.version" value="2.0.3"/>
```

Comment:
I missed this before but looks like https://www.eclipse.org/downloads/download.php?file=/jakartaee/transactions/2.0/jakarta-transactions-tck-2.0.1.zip was the last Transactions TCK to be promoted.  So I think we can just stick with the release/tools/jta.xml deliverable.tck.version = 2.0.2 value.

Reply:
Everything else looks good.

Reply:
thanks @scottmarlow, someone has incremented release version without an actual release.I have removed increment of the JTA release.

Reply:
I have reverted the version increment for JTA, also
found JTA 2.0.2 TCK bundle service release at - https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee10/promoted/eftl/